src/main/flight/mixer_tricopter.h

```diff
@@ -73,4 +73,82 @@ int16_t triGetMotorCorrection(uint8_t motorIndex);
  */
 _Bool triEnableServoUnarmed(void);
 
+#ifdef USE_SERVOS
```

Comment:
Wow you are quick.
Thanks for the input. I think your second pattern is really nice. No extra files.
(Now time to do my day job..)